css/main.css

```diff
@@ -0,0 +1,76 @@
+@font-face {
+  font-family: digital;
+  src: url(fonts/digital-7.ttf);
+}
+
+
+
+body {
+    background: black;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+input {
+    background-color: lightgray;
+    height: 50px;
+    width: 700px;
+    margin-top: 100px;
+    margin-bottom: 50px;
+    border-radius: 15px;
+    
+    font: 60px digital;
```

Comment:
Good practice to add font fallbacks in case main font doesn't exist or doesn't download. Example: `font-family: "Roboto", Arial, Helvetica, sans-serif;`